modules/flowable-json-converter/src/test/java/org/flowable/editor/language/UsertTaskWithSpacedExtensionIdConverterTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.flowable.editor.language;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.flowable.bpmn.model.BpmnModel;
+import org.flowable.bpmn.model.FlowElement;
+import org.flowable.bpmn.model.UserTask;
+import org.junit.jupiter.api.Test;
+
+public class UsertTaskWithSpacedExtensionIdConverterTest extends AbstractConverterTest {
+
+  @Test
+  public void convertJsonToModel() throws Exception {
+    BpmnModel bpmnModel = readJsonFile();
+    validateModel(bpmnModel);
+  }
+
+  @Test
+  public void doubleConversionValidation() throws Exception {
+    BpmnModel bpmnModel = readJsonFile();
+    bpmnModel = convertToJsonAndBack(bpmnModel);
+    validateModel(bpmnModel);
+  }
+
+  @Override
+  protected String getResource() {
+    return "test.usertaskmodelwithspacedextensionid.json";
+  }
+
+  private void validateModel(BpmnModel bpmnModel) {
+    FlowElement flowElement = bpmnModel.getMainProcess().getFlowElement("usertask", true);
+    assertThat(flowElement).isInstanceOf(UserTask.class);
+    UserTask userTask = (UserTask) flowElement;
+    assertThat(userTask.getExtensionElements().containsKey("group-info-name-human_resources"))
+        .isTrue();
```

Comment:
You might try simplifying this to (untested):
`assertThat(userTask.getExtensionElements()).containsKey("group-info-name-human_resources");`

Reply:
Done